### PR TITLE
add system tests workflow

### PR DESCRIPTION
**Pull request by stewartie4**
Tasks to complete before merging PR:
- [ ] [Run system tests](https://github.com/0chain/zboxcli/actions/workflows/system_tests.yml) :clipboard:

### CODE REVIEW

**Review by cnlangzi — APPROVED** [review]
LGTM

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  build-linux:
+    runs-on: [self-hosted, load-test]
+    needs: unit-test
+    steps:
+      - name: Setup go 1.17
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+
+      - name: Clone 0Box
```
Comment (kushthedude) [inline .github/workflows/tests.yml:36]: zbox?

**Review by kushthedude — DISMISSED** [review]
Some minor neat

File: .github/workflows/tests.yml (hunk 1 above)
Comment (rizary) [reply .github/workflows/tests.yml:36]: ~~the name of zboxcli binary is zbox, not zboxcli~~.

**Pull request merged**